src/client/components/stage-details/stage-details.stories.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import StageDetails from './stage-details.component';
+import data from './crop_stage_default_values.json';
+
+export default {
+  component: StageDetails,
+  title: 'Accordion Options',
+};
+
+const cropID = 1;
+
+const parametersAndLabels = {
+  temperature: 'Temperature (°C)',
+  humidity: 'Humidity (g/m3)',
+  ph: 'PH',
+  ec: 'EC (ppm)',
+  // eslint-disable-next-line @typescript-eslint/camelcase
+  water_level: 'Water level (cm)',
+};
```

Comment:
I am not very happy with variable or function names that include `And` 😁 

In my opinion, these are not parameters and labels but rather `units` so I would rename it like this and make it more clear it is something constant:
```javascript
const UNITS = {
  TEMPERATURE: 'Temperature (°C)',
  HUMIDITY: 'Humidity (g/m3)',
  PH: 'PH',
  EC: 'EC (ppm)',
  // eslint-disable-next-line @typescript-eslint/camelcase
  WATER_LEVEL: 'Water level (cm)',
};
```


The last thing is I think this should be defined in a place where it can be reusable and not in the story 👍

Reply:
Those are all great points, thanks! 💯 

If it's not much of an issue, a name like `PARAMETERS` could be better for clarity since that's what the company calls them (see [comment](https://github.com/HackYourFuture-CPH/FP-class11/pull/19#discussion_r398820603)). 

I understand it's good to write the constants using uppercase, but in this case, the keys correspond to the actual values in Seasony's data output, so perhaps converting them to lowercase any time they're used would add to the confusion? E.g. from [here](https://github.com/HackYourFuture-CPH/FP-class11/pull/19/files#diff-b9e92b9fb13172277b5dad8d319f74eb):

```
  {
    "id": "1",
    ....
    "parameter": "temperature",
    ...
  },
  {
    "id": "2",
    ....
    "parameter": "humidity",
    ...
  },
```

I'll relocate the object to inside the component's file. 👍